src/app/factories/spcp-myinfo.factory.js

```diff
@@ -66,11 +66,6 @@ const spcpFactory = ({ isEnabled, props }) => {
       myInfoConfig.appId = myInfoPrefix + myInfoConfig.singpassEserviceId
       myInfoConfig.mode = process.env.MYINFO_CLIENT_CONFIG
       myInfoGovClient = new MyInfoGovClient(myInfoConfig)
-    } else if (config.isDev && process.env.MYINFO_APP_KEY) {
-      myInfoConfig.appId = 'STG2-' + myInfoConfig.singpassEserviceId
-      myInfoConfig.privateKey = process.env.MYINFO_APP_KEY
-      myInfoConfig.mode = 'stg'
```

Comment:
do we no longer need a `stg` mode?

Reply:
we do, but it's set via the env var `MYINFO_CLIENT_CONFIG`, which is set to `stg` in our staging environment. `MYINFO_APP_KEY` is unused.

Reply:
will this break the development integration with mockpass?

Reply:
so if I'm not wrong, the purpose of this is to have a dev environment that talks to MyInfo dev instead of MyInfo mockpass? MyInfo has 4 different environments if I'm not wrong; Prod, staging, dev and mockpass. Am i correct @liangyuanruo ?

Reply:
nope! we don't use this env var in our dev environment. we only use `MYINFO_CLIENT_CONFIG`.

Reply:
isn't the purpose of this just so that you can specify a key directly in the env vars instead of specifying a file path? why would it be listed as "deprecated" in our documentation if it's required for something?

Reply:
@arshadali172 do you mean the myinfo-gov-client mode? that mode is specified using `MYINFO_CLIENT_CONFIG`